src/napi.zig

```diff
@@ -2,179 +2,154 @@ const std = @import("std");
 const napigen = @import("napigen");
 const lib = @import("root");
 const platform = @import("platform.zig");
-const Style = @import("style.zig").Style;
-const Node = lib.Node;
-const Document = lib.Document;
-const Element = lib.Element;
-const css = @import("css.zig");
 
 // globals
-var allocator = std.heap.c_allocator;
 var window: *platform.Window = undefined;
 var document: *lib.Document = undefined;
 var renderer: lib.Renderer = undefined;
-var js: napigen.Context = undefined;
-var handleEventRef: napigen.napi_ref = undefined;
-var uv_loop: ?*napigen.uv_loop_s = undefined;
-var prepare_handle: uv_prepare_t = undefined;
-var idle_handle: uv_idle_t = undefined;
-var awaker: std.Thread = undefined;
-
-const JsInitOpts = struct {
-    handleEvent: napigen.napi_value,
-};
-
-fn init(opts: JsInitOpts) !napigen.napi_value {
+
+comptime {
+    napigen.defineModule(&initModule);
+}
+
+fn initModule(js: *napigen.JsContext, exports: napigen.napi_value) !napigen.napi_value {
+    try js.setNamedProperty(exports, "init", try js.createFunction(&init));
+
+    const defs = .{
+        .Node = .{ .parent_node, .first_child, .next_sibling, .appendChild },
+        // .Element = .{ .local_name, .style, .getAttribute, .setAttribute, .removeAttribute },
```

Comment:
this will be a problem (missing methods, etc.) - you need to use the previous version with napi_register_module_v1